managed/CustomGroup_Currency_Exchange_rates.mgd.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+return [
+  [
+    'name' => 'CustomGroup_Currency_Exchange_rates',
+    'entity' => 'CustomGroup',
+    'cleanup' => 'unused',
+    'update' => 'always',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'Currency_Exchange_rates',
```

Comment:
it is better for the name to have all the characters to be small letter. Same applies for the custom fields below.